packages/manageState/src/index.ts

```diff
@@ -0,0 +1,50 @@
+/// <reference path="../../../types/wallet.d.ts" />
+
+// bs comment out 050622
+// import openrpcDocument from './openrpc.json';
+
+// eslint-disable-next-line import/unambiguous
+wallet.registerRpcMessageHandler(async (_originString, requestObject) => {
+  let state = (await wallet.request({
+    method: 'snap_manageState',
+    params: ['get'],
+  })) as { testState: string[] } | null;
+
+  if (!state) {
+    state = { testState: [] };
+    // initialize state if empty and set default data
+    await wallet.request({
+      method: 'snap_manageState',
+      params: ['update', state],
+    });
+  }
+
+  switch (requestObject.method) {
+    //  bs comment out 050622
+    //  case 'rpc.discover':
+    //    return openrpcDocument;
+    case 'storeTestData':
+      state.testState.push(requestObject.params[0]);
+      await wallet.request({
+        method: 'snap_manageState',
+        params: ['update', state],
+      });
+      return true;
+    case 'retrieveTestData':
+      //    return state;
+      const persistedData = await wallet.request({
+        method: 'snap_manageState',
+        params: ['get'],
+      });
+      return persistedData;
```

Comment:
The linter forbids declaring variables in `case` blocks unless you wrap them like `case CASE: { /* content goes here */ }`. However, you don't even need to declare a variable in this case:
```suggestion
    case 'retrieveTestData':
      return await wallet.request({
        method: 'snap_manageState',
        params: ['get'],
      });
```

Reply:
Oh sweet, thanks!